JWBestPracticeApps/AirPlay/AppDelegate.swift

```diff
@@ -7,17 +7,36 @@
 //
 
 import UIKit
+import AVFoundation
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        
+        setUpBackgroundAudio()
+        
         return true
     }
     
+    // Set up the app for playback. Necessary for AirPlay to run smoothly.
+    func setUpBackgroundAudio() {
+        let audioSession = AVAudioSession.sharedInstance()
+        
+        do {
+            try audioSession.setCategory(AVAudioSession.Category.playback)
+        } catch {
+            print("Setting category to AVAudioSessionCategoryPlayback failed.")
+        }
+        
+        do {
+            try audioSession.setActive(true, options: .notifyOthersOnDeactivation)
```

Comment:
`.notifyOthersOnDeactivation` must be used when you are deactivating the audio session only.

For example `try audioSession.setActive(false, options: .notifyOthersOnDeactivation)` in the `applicationWillTerminate:` delegate.